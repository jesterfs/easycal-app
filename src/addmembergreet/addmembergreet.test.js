import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import AddMemberGreet from './addmembergreet'

import renderer from 'react-test-renderer';

describe('AddMemberGreet component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <AddMemberGreet />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <AddMemberGreet />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});