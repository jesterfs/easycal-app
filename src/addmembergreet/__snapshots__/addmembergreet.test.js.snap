// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMemberGreet component renders the UI as expected 1`] = `
<div
  className="AddMemberGreet, greetgroup"
>
  <div
    className="item"
  >
    <h2>
      Add a Team Member
    </h2>
    <p>
      The invitee will have to finish the sign-up process.
    </p>
  </div>
  <div
    className="item"
  >
    <form
      className="addMemberForm"
      onSubmit={[Function]}
    >
      <div>
        <label
          htmlFor="memberName"
        >
          Full Name
        </label>
        <input
          id="memberName"
          name="memberName"
          placeholder="Full Name"
          type="text"
        />
      </div>
      <div>
        <label
          htmlFor="memberEmail"
        >
          Email
        </label>
        <input
          id="memberEmail"
          name="memberEmail"
          placeholder="Email"
          type="email"
        />
      </div>
      <div>
        <label
          htmlFor="selectMemberCalendars"
        >
          Select Calendars:
        </label>
        <br />
        <select
          id="selectMemberCalendars"
          multiple={true}
          name="selectMemberCalendars"
        />
      </div>
      <button
        className="addbtn"
        type="submit"
      >
        Send Invite
      </button>
    </form>
  </div>
</div>
`;
